web-src/src/admin/components/scripts-config/ParameterConfigForm.vue

```diff
@@ -73,6 +73,12 @@
                  title="Excluded files"
                  @error="handleError('Excluded files', $event)"/>
     </div>
+    <div v-if="selectedType === 'text' || selectedType === undefined" class="row">
```

Comment:
May be you could combine it with the max length row?

Reply:
You mean, move both `regex_pattern` & `regex_description` textfields under the same `div` with `max_length`?

Reply:
yep